### PR TITLE
Readme: typo, grammar and semantic fixes

### PR DESCRIPTION
**Pull request by ggreif**
It is past midnight, so I might have gotten the "reasonable" wrong.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,11 +58,11 @@ After this, find the binary at `target/release/quill`.
 
 `quill` is a very critical link in the workflow of the management of valuable assets.
 `quill`'s code must stay clean, simple, readable and leave no room for ambiguities, so that it can be reviewed and audited by anyone.
-Hence, if you would like to propose a change, please adhere to the following principals:
+Hence, if you would like to propose a change, please adhere to the following principles:
 
 1. Be concise and only add functional code.
 2. Optimize for correctness, then for readability.
-3. Avoid adding dependencies at all costs unless it's completely unreasonable.
+3. Avoid adding dependencies at all costs unless it's completely reasonable.
```
Comment (chmllr) [inline README.md:65]: Well, I meant "unless it's completely _unreasonable to avoid_ a library" :) 

It's almost always "reasonable" to not reinvent the wheel, except in security relevant code, where every dependency is an attack vector. So if this sentence is misleading, we need another formulation, but this new version is not exactly what I meant.

File: README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ To display the signed message in human-readable form:
 
     quill send --dry-run <path-to-file>
 
-`quill` could be used on an online computer to send any signed transactions:
+`quill` could be used on an online computer to send any signed transaction:
```
Comment (chmllr) [inline README.md:20]: But why is it singular now? The file passed to `quill` can literally contains multiple signed transactions, e.g. when you stake a neuron. Maybe we should just drop "any" from the original text?

Comment (ggreif) [reply README.md:20]: I think "any" with singular still means there can be several, but you don't really care what they are like. https://english.stackexchange.com/questions/111444/any-individual-or-any-individuals

Let's bounce this by a native speaker. @jwiegley?

File: README.md (hunk 1 above)
Comment (ggreif) [reply README.md:65]: Haha, see? Past midnight delusions. I understood "Never add dependencies, unless you find a totally unreasonable one, yay!" :-)

Comment (ggreif) [reply README.md:65]:
```suggestion
3. Avoid adding dependencies at all costs unless it's completely unreasonable to do so.
```

**Review by chmllr — DISMISSED** [review]

**Comment by chmllr** [comment]
Thanks so much @ggreif !

**Review by chmllr — APPROVED** [review]

**Pull request merged**